.gitignore

```diff
@@ -1,35 +1,16 @@
-vendor
-
 # Binaries for programs and plugins
-*.exe
-*.exe~
-*.dll
-*.so
-*.dylib
-bin/*
-testbin/*
-Dockerfile.cross
-
-# Test binary, build with `go test -c`
-*.test
+bin/
 
-# Output of the go coverage tool, specifically when used with LiteIDE
+# Output of the go coverage tool
 *.out
 
 # Release output
-dist/**
 operator-controller.yaml
-install.sh
```

Comment:
Don't get generate `install.sh` as part of `make run|e2e|release`? And goreleaser creates the `dist` directory when we build binaries and run the release. So I think these lines need to stay.